packages/vapor/scss/components/badge.scss

```diff
@@ -37,27 +37,27 @@
     }
 
     &.mod-critical {
-        --color: var(--critical-100);
-        --border-color: var(--critical-20);
-        --background-color: var(--critical-20);
+        --color: var(--semantic-critical-100);
```

Comment:
I think it would be wiser to change the `--critical-*`, `--warning-*`, `--information-*` and `--success-*` colors instead if it makes sense

Reply:
> While at it do you think it would be possible to fix the alignment issue? It seems too close to the bottom.
> ![image](https://user-images.githubusercontent.com/35579930/114732907-e6abfa00-9d10-11eb-85d3-a5f5ff1309e7.png)

yes :smile: :yum: 
https://coveo.slack.com/archives/GLS6Z9RPD/p1618410483055600?thread_ts=1618337997.050200&cid=GLS6Z9RPD

Reply:
> I think it would be wiser to change the `--critical-*`, `--warning-*`, `--information-*` and `--success-*` colors instead if it makes sense

I did not want to change the colors to much since they are used a lot everywhere :sweat_smile: 
If those are the final* colors, then I think we need a user story to change them all :thinking: 

*_subject to change_  :rofl:

Reply:
Yes I understand the concern. I think it would be worth confirming with the designers that this is what we want, because reverting back to variables with specific color names feels like a regression in terms of scalability. The goal what to have variable names that can stand theme changes.

Reply:
Yeah i'll ask :+1:

Reply:
I've done some tests and for information-XX there's other components that the colors is changing also. I feel like this is getting out of scope :see_no_evil: 

I'll open a Jira to get this sorted out properly, I don't want to do it more than once :laughing:

Reply:
https://coveord.atlassian.net/browse/ADUI-6881

Reply:
Sounds good!